open-company/open-company-auth
# Fix active user invite

BUG: when an active user is invited as admin we don't set him as admin but simply add him to the team.

To test:
- signup via email creating org1 with user1 (don't verify your email)
- signup via email creating org2 with user2 (verify your email)
- now with user1 invite user2 to org1 as admin
- [x] user2 is an admin in org1?
- now with user2 invite user1 to org2 as admin
- [x] user1 is an admin in org2?
- now add an email domain to org1
- signup with user3 using that email domain
- [x] are you locked out pending email verification?
- now with user2 invite user3 to org2
- try logging in with user3
- [x] still locked out pending email verification?
